index.html

```diff
@@ -20,10 +20,6 @@
     <!-- Add to home screen for Windows -->
     <meta name="msapplication-TileImage" content="/static/img/icons/msapplication-icon-144x144.png">
     <meta name="msapplication-TileColor" content="#000000">
-    <% for (var chunk of webpack.chunks) {
```

Comment:
Didn't realise this was preloading all the chunks, which is why I thought code splitting wasn't working.